invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/request_actions/InvitationActionButtons.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import { Button } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_communities/i18next";
+import PropTypes from "prop-types";
+import {
+  RequestActionContext,
+} from "@js/invenio_requests";
+
+export class ActionButtons extends Component {
+  static contextType = RequestActionContext;
+
+  render() {
+    return (
+      <Button
+        as="a"
+        target="_blank"
+        size="medium"
+        content={i18next.t("View")}
+        href={this.context.linkExtractor.self_html}
```

Comment:
When I read this, it looks a bit weird... there is a lot of assumption here that such property has actually been set somehow.
It looks like that we are passing to this cmp a prop `request` (missing in the propTypes?): can we get the link from there instead?